<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPParam</class>
 <widget class="QWidget" name="CPParam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="plainTextEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="prefixLabel">
     <property name="text">
      <string>PP-Prefix</string>
     </property>
     <property name="buddy">
      <cstring>prefixEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="suffixLabel">
     <property name="text">
      <string>PP-Suffix</string>
     </property>
     <property name="buddy">
      <cstring>suffixEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="prefixEdit"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="suffixEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="nlLabel">
     <property name="text">
      <string>PP-Nonlocality</string>
     </property>
     <property name="buddy">
      <cstring>nlEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="nlEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
